pgo/cmd/auth.go

```diff
@@ -66,8 +66,19 @@ func UserHomeDir() string {
 }
 
 func parseCredentials(dat string) msgs.BasicAuthCredentials {
+	lines := strings.Split(strings.TrimSpace(dat), "\n")
```

Comment:
May be worth considering a different formulation that handles newline conventions of different systems. (Notably, windows' convention is '\r\n' and would bite a user that edited their file on windows and binary copied over to the system)